virtualcluster/pkg/syncer/util/featuregate/gate.go

```diff
@@ -68,6 +68,10 @@ const (
 	// ClusterVersionPartialUpgrade is an experimental feature that allows the cluster provisioner
 	// to apply ClusterVersion updates if VirtualCluster object is requested it
 	ClusterVersionPartialUpgrade = "ClusterVersionPartialUpgrade"
+
+	// ResourceNoSync is an experimental feature that gives tenant the capability
+	// of not syncing certain resources to super cluster.
+	ResourceNoSync = "ResourceNoSync"
```

Comment:
Sorry for being paranoid, I wish the feature name exactly expresses its intention. Hence I suggest naming it to
"TenantAllowResourceNoSync"

Reply:
This name is better. Thanks